aws/policy/security-services.yaml

```diff
@@ -22,12 +22,14 @@ Statement:
       - iam:DetachRolePolicy
     Resource:
       - arn:aws:iam::{{ aws_account_id }}:role/ansible-test-sts-*
+      - arn:aws:iam::{{ aws_account_id }}:role/ansible_lambda_role
```

Comment:
What's this for and the change below?

Reply:
I'm going to have to go with "something I was trying for some reason and forgot to remove". `aws_eks` passes on dev without these so away they go.